Dockerfile

```diff
@@ -2,13 +2,14 @@ FROM madnificent/ember:3.17.0 as ember
 MAINTAINER C.S.V. Alpha <ict@csvalpha.nl>
 
 ARG DEPLOY_TARGET='production'
+ARG BUILD_HASH='unknown'
```

Comment:
Shouldn't these (also DEPLOY_TARGET) be ENVs because they're environment variables in the docker-compose.buildkite.yml too?

Reply:
What we do is set it as environment in docker-compose for buildkite, and then use that environment variable in the build step for this docker image. The difference is that a build argument is only used during build, where the environment variable is used and set during execution

Reply:
That makes a ton of sense now, thanks for the explanation 👍🏼